ert_gui/simulation/run_dialog.py

```diff
@@ -90,7 +86,7 @@ def __init__(self, run_model, parent):
         self.plot_tool.setParent(None)
         self.plot_button = QPushButton(self.plot_tool.getName())
         self.plot_button.clicked.connect(self.plot_tool.trigger)
-        self.plot_button.setEnabled(ert is not None)
```

Comment:
Is there no chance that ert = run_model.ert() returns None?

Reply:
Actually I'm not sure - but from my understanding it's currently impossible for ert to be None - but i may be mistaken here... previously an assertion would make assertion error if run_model is not a BaseRunModel, the same check is used in if where ert was set... as far as i understand the if could never fail previously. This check is removed - the whole ert assignment is removed - and now the global ERT is used where ert has to be set when initializing the code - either through GUI or CLI

```
        assert isinstance(run_model, BaseRunModel)
        self._run_model = run_model

        ert = None
        if isinstance(run_model, BaseRunModel):
            ert = run_model.ert()
```